tensorboard/webapp/util/colors.ts

```diff
@@ -73,3 +73,77 @@ export const DEFAULT_PALETTE: ColorPalette = {
     darkHex: '#3b3b3b',
   },
 };
+
+// Default color scheme from Polymer-based TensorBoard.
+// A colorblind-friendly palette designed for TensorBoard by Paul Tol
+// (https://personal.sron.nl/~pault/).
+export const classic: ColorPalette = {
+  id: 'classic',
+  name: 'Classic',
+  colors: [
+    {name: 'Orange', lightHex: '#ff7043', darkHex: '#ff7043'},
+    {name: 'Blue', lightHex: '#0077bb', darkHex: '#0077bb'},
+    {name: 'Red', lightHex: '#cc3311', darkHex: '#cc3311'},
+    {name: 'Cyan', lightHex: '#33bbee', darkHex: '#33bbee'},
+    {name: 'Magenta', lightHex: '#ee3377', darkHex: '#ee3377'},
+    {name: 'Teal', lightHex: '#009988', darkHex: '#009988'},
+    {name: 'Gray', lightHex: '#bbbbbb', darkHex: '#bbbbbb'},
+  ],
+  inactive: {
+    name: 'Light gray',
+    lightHex: '#e0e0e0', // Gray 300
+    darkHex: '#3b3b3b',
+  },
+};
+
+export const googleStandard: ColorPalette = {
+  id: 'goog_standard',
+  name: 'Google Standard',
+  colors: [
+    {name: 'Google Red 500', lightHex: '#db4437', darkHex: '#db4437'},
+    {name: 'Deep Orange 400', lightHex: '#ff7043', darkHex: '#ff7043'},
+    {name: 'Google Yellow 500', lightHex: '#f4b400', darkHex: '#f4b400'},
+    {name: 'Google Green 500', lightHex: '#0f9d58', darkHex: '#0f9d58'},
+    {name: 'Teal 700', lightHex: '#00796b', darkHex: '#00796b'},
+    {name: 'Cyan 600', lightHex: '#00acc1', darkHex: '#00acc1'},
+    {name: 'Google Blue 500', lightHex: '#4285f4', darkHex: '#4285f4'},
+    {name: 'Indigo 400', lightHex: '#5c6bc0', darkHex: '#5c6bc0'},
+    {name: 'Purple 400', lightHex: '#ab47bc', darkHex: '#ab47bc'},
+  ],
+  inactive: {
+    name: 'Light gray',
+    lightHex: '#e0e0e0', // Gray 300
+    darkHex: '#3b3b3b',
+  },
+};
+
+export const colorBlindAssist3: ColorPalette = {
+  id: 'color_blind_assist_3',
+  name: 'Color Blind Assist',
```

Comment:
This name is kind of confusing now that it's not in the context of the others in palettes.ts.  It's actually less-colorblind-friendly than the classic scheme probably is, since the tradeoff made here is lower contrast between colors but a higher number of colors.  Maybe it would be better to call this "Classic Extended" or just "Extended" or something?